pkg/ddc/jindo/runtime_info_test.go

```diff
@@ -26,6 +26,11 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
+// newJindoEngineRT creates a new JindoEngine instance for testing.
+// It initializes the engine with a default JindoRuntime and a null logger.
+// When withRuntimeInfo is true, it also builds and assigns the runtimeInfo using the
+// provided name, namespace, and common.JindoRuntime type. This helper is used in tests to
+// simulate different runtime configurations.
 
```

Comment:
No blank line between comments and method. Leave one blank line between comments and import.